README.md

```diff
@@ -1,16 +1,16 @@
-# RpcEngine
+# json-rpc-engine
 
-a tool for processing JSON RPC
+A tool for processing JSON-RPC requests and responses.
 
-### usage
+## Usage
 
 ```js
 const RpcEngine = require('json-rpc-engine')
 
 let engine = new RpcEngine()
 ```
 
-Build a stack of json rpc processors by pushing in RpcEngine middleware.
+Build a stack of JSON-RPC processors by pushing middlewares to the engine.
```

Comment:
I've noticed the plural "middlewares" being used in a few places now, and I'm not sure it's correct? "middleware" is already plural.

```suggestion
Build a stack of JSON-RPC processors by pushing middleware to the engine.
```

Reply:
This is done in various other places in this README as well